ir/inst_other.go

```diff
@@ -674,3 +674,48 @@ func (inst *InstCleanupPad) LLString() string {
 	}
 	return buf.String()
 }
+
+// ~~~ [ freeze ] ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+// InstFreeze is an LLVM IR freeze instruction.
+type InstFreeze struct {
+	// Name of local variable associated with the result.
+	LocalIdent
+
```

Comment:
```suggestion
	// Operand.
```

This is the same as the comment we use for the unary instruction `ir.InstFNeg.X`.